package.json

```diff
@@ -46,6 +46,7 @@
     "swarm-sasstools": "^3.0.312"
   },
   "dependencies": {
+    "@storybook/addon-actions": "false3.4.2",
```

Comment:
for some reason I was getting some errors around `@storybook/addon-actions` and could not run storybook, so I installed it using: `yarn add @storybook/addon-actions`. Don't know why there is a `false` in front of the versioning.
![screen shot 2018-04-18 at 2 40 13 pm](https://user-images.githubusercontent.com/7966507/38960556-f05e6230-4332-11e8-8912-8c965b3a3445.png)